File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromInternalToAlphaTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.PublishingService;

public class PromoteFromInternalToAlphaTask extends BaseTask {
	
	public PromoteFromInternalToAlphaTask() {
		setDescription("Promote a current internal to alpha");
	}

	@Override
	protected void onExecute(App app) {
		new PublishingService().promoteFromInternalToAlpha(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromInternalToAlphaTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class PromoteFromInternalToAlphaTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().promoteFromInternalToAlpha(app)
    }

    init {
        description = "Promote a current internal to alpha"
    }
}
